modules/nw-metallb-configure-return-traffic-proc.adoc

```diff
@@ -40,37 +42,48 @@ spec:
   maxUnavailable: 3
   desiredState:
     interfaces:
-      - name: ens4vrf <3>
-        type: vrf <4>
-        state: up
-        vrf:
-          port:
-            - ens4 <5>
-          route-table-id: 2 <6>
-    routes: <7>
+    - name: ens4vrf <3>
+      type: vrf <4>
+      state: up
+      vrf:
+        port:
+        - ens4 <5>
+        route-table-id: 2 <6>
+    - name: ens4 <7>
+      type: ethernet
+      state: up
+      ipv4:
+        address:
+        - ip: 192.168.130.130
+          prefix-length: 24
+        dhcp: false
+        enabled: true
+    routes: <8>
       config:
-        - destination: 0.0.0.0/0
-          metric: 150
-          next-hop-address: 192.168.130.1
-          next-hop-interface: ens4
-          table-id: 2
-    route-rules: <8>
+      - destination: 0.0.0.0/0
+        metric: 150
+        next-hop-address: 192.168.130.1
+        next-hop-interface: ens4
+        table-id: 2
+    route-rules: <9>
```

Comment:
can you add another block to route-rules
this is the ovn host-to-service masquerade subnet its sometimes used for external IP -> service natting and needs a rule for vrfs to ingress traffic 
```
- ip-to: 169.254.169.0/29
          priority: 998
          route-table: 254
```